src/main/java/github/weichware10/toolbox/codecharts/CodeCharts.java

```diff
@@ -0,0 +1,68 @@
+package github.weichware10.toolbox.codecharts;
+
+import github.weichware10.toolbox.gui.End;
+import github.weichware10.util.ToolType;
+import github.weichware10.util.Logger;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Logger' import. Should be before 'github.weichware10.util.ToolType'.